.github/workflows/lint.yml

```diff
@@ -25,3 +25,34 @@ jobs:
         uses: docker://rhysd/actionlint:latest
         with:
           args: "-color -verbose -shellcheck="
+
+  octoscan:
+    name: Octoscan
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Checkout Octoscan
+      uses: actions/checkout@v4
+      with:
+        repository: synacktiv/octoscan
+        path: octoscan
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.21'
+
+    - name: Install and build Octoscan
+      run: | #shell
+        cd octoscan
+        go mod tidy
+        go build
+
+    - name: Run Octoscan
+      run: | #shell
+        # Octoscan does not use exit codes to indicate success or failure, so we need to check the output
+        ./octoscan/octoscan scan . | tee octoscan-result.txt
+        test ! -s octoscan-result.txt
```

Comment:
```suggestion
        ! ./octoscan/octoscan scan .github/workflows/ | grep '^'
```

Reply:
The path needs to point to the repo root, not the workflows dir. Otherwise looks good.

Reply:
@szepeviktor It looks like `'^'` is actually grepping for the start of a line, not the literal `^` character. Needs the `-F` flag I think.

Reply:
![2024-08-10-17-32-47](https://github.com/user-attachments/assets/eb8410a4-89a5-4613-ab91-d549a54d894b)

Reply:
Going to leave this as-is 👍

Reply:
`-F` causes it take `^` literally!!

Reply:
carot is for the start of the line, meaning "anything"

Reply:
`test ! -s` does the same thing

Reply:
The above test fails for a non-empty output.
If you need an empty output remove `!`.

Reply:
All right.